snap/info.go

```diff
@@ -643,28 +644,69 @@ func (s *Info) CommonDataSaveDir() string {
 	return CommonDataSaveDir(s.InstanceName())
 }
 
+// Get home directories from system-params file.
+// Home directories are returned as a ',' seperated string
```

Comment:
Minor typo, "seperated"/"separated", mentioning as it is causing static checks to fail.

It's usually a good idea  to run tests locally using `./run-checks`, you can find great info about testing [here](https://github.com/snapcore/snapd/blob/master/HACKING.md#testing). You can find helpful information in [HACKING.md](https://github.com/snapcore/snapd/blob/master/HACKING.md) and [CONTRIBUTING.md](https://github.com/snapcore/snapd/blob/master/CONTRIBUTING.md)